packages/react-chat/src/utils/stylesheet.ts

```diff
@@ -22,9 +22,9 @@ export const useResolveAssistantStyleSheet = (assistant?: AssistantOptions, shad
   const [isStyleSheetResolved, setStyleSheetResolved] = useState(false);
 
   useEffect(() => {
-    if (!assistant || isStyleSheetResolved || !shadowRoot) return;
```

Comment:
with this logic it only renders the `<ChatEmbed>` and `<ChatWidget>` if using the shadowDOM